en/tools/config-catalog-generator-cc/data/adapter/configs.json

```diff
@@ -0,0 +1,200 @@
+{
+  "configs": [
+    {
+      "title": "Server",
+      "options": [
+        {
+          "name": "server",
+          "required": true,
+          "description": "This includes configurations required for configuring the deployment parameters that are used for identifying the Choreo Connect Adapter REST APIs.",
+          "params": [
+            {
+              "name": "enabled",
+              "type": "boolean",
+              "required": true,
+              "default": "true",
+              "possible": "true, false",
+              "description": "Enable/Disable Choreo Connect Adaptor REST API."
```

Comment:
Typo across many descriptions with the usage of the word `Adaptor`. Replace all instances with `Adapter`.